numpy/linalg/tests/test_regression.py

```diff
@@ -90,6 +90,20 @@ def test_svd_no_uv(self):
                 assert_equal(np.linalg.matrix_rank(a), 1)
                 assert_array_less(1, np.linalg.norm(a, ord=2))
 
+    def test_eig_vs_eigh_above_560(self):
+        # gh-6896
```

Comment:
Could you add a bit more documentation about what is being tested and where it currently fails?